Tweet-311-Denver/tweet_311_denver_ui
# Fix map error

## What does this PR do?
- Fixes the a bug in maps, when you clicked add location, with no location selected an error was thrown
- Add new icons to validate when a location has been selected 
- Adds some new styling to the map

## How to test?
- Pull down
- Run `npm start`
- Enter maps component, do not select location, click `add location` and see there is no more error
- You can also see the icons change when you toggle a location

## Issues:
- This PR should close #30 
- I have updated the board 

## Screenshots:
![Screen Shot 2020-02-24 at 1 23 14 PM](https://user-images.githubusercontent.com/48968224/75188130-d13cd080-5708-11ea-9c73-1f902dc6eeae.png)
![Screen Shot 2020-02-24 at 1 23 24 PM](https://user-images.githubusercontent.com/48968224/75188141-d732b180-5708-11ea-889e-a4eff90f9628.png)

## Notes:
- Not much really else to add here. The border around the map is hard to style. I want to just add a borderBottom width and color but it's not letting me. Something for later

Thanks!
@cammac60
